app/views/planning_applications/assessment/local_policies/_table.html.erb

```diff
@@ -25,7 +25,8 @@
           <%= policy_area.assessment %>
         </td>
         <td class="govuk-table__cell govuk-!-padding-top-4">
-          <%= button_to "Delete", planning_application_assessment_local_policy_area_path(@planning_application, policy_area), data: { confirm: "Area you sure?" }, method: :delete, class: "govuk-button govuk-button--warning" %>
+          <%= link_to "Edit", planning_application_assessment_local_policy_area_path(@planning_application, policy_area) %>
+          <%= link_to "Remove", planning_application_assessment_local_policy_area_path(@planning_application, policy_area), data: { confirm: "Are you sure?" }, method: :delete %>
```

Comment:
Are we using this "Are you sure?" data confirm elsewhere? I'm just thinking I did something like this recently so if we are, I need to change it 😬

Reply:
Agreed to remove after consulting Laura.